python/ray/_private/utils.py

```diff
@@ -1964,3 +1970,22 @@ def validate_node_labels(labels: Dict[str, str]):
                 f"{ray_constants.RAY_DEFAULT_LABEL_KEYS_PREFIX}. "
                 f"This is reserved for Ray defined labels."
             )
+
+
+def pasre_pg_formatted_resources_to_origin(pg_formatted_resources):
+    origin_resources = {}
+    if pg_formatted_resources is None:
+        return origin_resources
+
+    for key, value in pg_formatted_resources.items():
+        result = PLACEMENT_GROUP_RESOURCE_PATTERN.match(key)
+        if result and len(result.groups()) == 2:
+            origin_resources[result.group(1)] = value
```

Comment:
Can you do 

```
if result.group(1) not in origin_resources:
    origin_resources[result.group(1)] = 0
origin_resources[result.group(1)] += value
``` 

instead? I know we cannot assign CPU & pg CPU from the same worker, but I think it is better not making that assumption

Reply:
I think logic after fixed is better.
![image](https://github.com/ray-project/ray/assets/11072802/82dd9164-e9ed-4ea1-af79-58da2047f2f8)

Reply:
hmm does it work if we have this kind of case?

```
CPU_group_4da1c24ac25bec85bc817b258b5201000000: 2.0
CPU_group_0_4da1c24ac25bec85bc817b258b5201000000: 1.0
CPU_group_1_4da1c24ac25bec85bc817b258b5201000000: 1.0
```

Reply:
It is impossible to have such a scenario where aTask/Actor is both assigned to two bundles.